src/main/java/com/codurance/guru/craftspeople/exceptions/CraftspersonDoesntExistException.java

```diff
@@ -0,0 +1,4 @@
+package com.codurance.guru.craftspeople.exceptions;
+
+public class CraftspersonDoesntExistException extends Throwable {
```

Comment:
is this an appropriate exception to extend from?

Reply:
fixed